Pricing & Settings/One Agent/LinkedTemplates/Solutions-deployIfNotExists.json

```diff
@@ -0,0 +1,73 @@
+{
```

Comment:
The requirement for Solution as part of the One Agent template is a known bug that is being worked on (ETA should be ~July). I fine with adding this now and removing it after. @bmanikm , does this make sense to you ?

Reply:
Yes, this needs to be removed once AMA enables the LA solutions automatically with DCR

---
In reply to: [439824405](https://github.com/Azure/Azure-Security-Center/pull/135#discussion_r439824405) [](ancestors = 439824405)